kong/status/ready.lua

```diff
@@ -0,0 +1,88 @@
+local declarative = require "kong.db.declarative"
+local constants = require "kong.constants"
+
+local tonumber = tonumber
+local kong = kong
+
+local get_current_hash = declarative.get_current_hash
+
+
+local worker_count = ngx.worker.count()
+local kong_shm     = ngx.shared.kong
+
+local ngx_log        = ngx.log
+local ngx_WARN       = ngx.WARN
+
+local is_traditional = kong.configuration.database ~= "off"
+local is_control_plane = kong.configuration.role == "control_plane"
```

Comment:
What does the endpoint mean to CP? CP does not serve any traffic, could it be always `200 OK`?

Reply:
we should use same check logic for all mode, i tried and but found it need some fix

Reply:
you are right we should skip check for cp and return ok